### DISCUSSION TITLE
Deviating Permissions for Package and Repository

### DISCUSSION THREAD

**Discussion by saltcreek-alex** [community/community#24790] [Apps, API and Webhooks]
<p>Our team is in the process of switching to Github and using the GPR for deploying our packages. <br>
The documentation states: </p>
<p>Manage permissions in one place</p>
<p>Packages in GitHub inherit the permissions of the repository, and you no longer need to manage third party solutions and sync team permissions across systems</p>
<p>This is generally a good idea and saves a lot of time, but we have certain situations where we want to share a private package with another development team, but not the source code itself. Since permissions are being synced, I wonder if it is possible to archieve the desired behaviour and how. </p>
<p>I have spent quite some time trying to figure out how to set up permissions for such a scenario, but so far nothing worked. Any ideas on how to archieve this? </p>
<p>Thank you in advance for any help!</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/saltcreek-alex">@saltcreek-alex</a>,</p>
<p>Thanks for being here! Accessibility of packages is tied to the visibility of the repository to which they are published. If a user has read access on the repository, then they will be able to install these packages, if they have write access on the repository then they will be able to publish new packages, as well as new versions of existing packages. If a repository is public, then the package is public, if the repository is private, the package is private.</p>
<p>As a work around you might want to consider creating a specific repository that is private in your organisation here and publishing packages to this.</p>

**Comment by saltcreek-alex** [comment]
<p>Hi <a class="mention" href="/andreagriffiths11">@andreagriffiths11</a>,</p>
<p>thank you for your response.<br>
We have already implemented this workaround and it works. I still feel like there should be a more convenient way to configure package and repository access independently from each other, perhaps in a future version.</p>
<p>Thank you for your time and help,<br>
Alex</p>
